### PR TITLE
Allow bind to wildcard (any) local address.

### PR DESCRIPTION
**Pull request by guusdk**
This commit allows the project to bind to any local address, by setting a property.

### CODE REVIEW

**Comment by guusdk** [comment]
Do you guys have a good way to test this? I've tried, but the snapshot-based versioning that's applied in Jitsi projects made me fail a couple of times (while trying to update this in three-levels-down dependency of my project).

**Comment by guusdk** [comment]
At the time of making this PR, another PR was already merged/accepted: https://github.com/jitsi/ice4j/pull/133 Unsure why jitsi-jenkins again asks for a CLA?

**Comment by damencho** [comment]
The CLA message is comming from a dumb plugin, we just need to whitelist you in every project, cause every project has its own settings.

**Comment by damencho** [comment]
Jenkins, add to whitelist.

File: src/main/java/org/ice4j/ice/harvest/AbstractTcpListener.java [hunk 1]
```diff
@@ -406,15 +406,44 @@ public void close()
     protected void init()
         throws IOException
     {
+        boolean bindWildcard = !StackProperties.getBoolean(
```
Comment (bgrozev) [inline src/main/java/org/ice4j/ice/harvest/AbstractTcpListener.java]: Why the negation?

Comment (guusdk) [reply src/main/java/org/ice4j/ice/harvest/AbstractTcpListener.java]: That's a mistake. Will fix asap.

File: src/main/java/org/ice4j/ice/harvest/AbstractUdpListener.java [hunk 2]
```diff
@@ -205,8 +205,24 @@ static String getUfrag(byte[] buf, int off, int len)
...
+                                        localAddress.getPort(),
+                                        localAddress.getTransport()
+                                );
+        }
+        else
+        {
+            this.localAddress = localAddress;
+        }
+
+        socket = new DatagramSocket( localAddress );
```
Comment (bgrozev) [inline src/main/java/org/ice4j/ice/harvest/AbstractUdpListener.java]: Do you want localAddress or this.localAddress here?

**Comment by bgrozev** [comment]
LGTM (pending the question about localAddress)

**Comment by guusdk** [comment]
Good catch. Now fixed.

**Comment by bgrozev** [comment]
Thanks!

**Pull request merged**